frontend/components/elements/TopicItem.vue

```diff
@@ -4,15 +4,23 @@
             <template v-if="!editing">
                 <BButton
                     size="sm"
-                    variant="danger"
+                    variant="outline-danger"
                     class="float-right ml-2"
-                    v-confirm="{ action: deleteTopic, text: 'Are you sure you want to delete this topic?' }">Delete</BButton>
+                    v-confirm="{ action: deleteTopic, text: 'Are you sure you want to delete this topic?' }">
```

Comment:
It is not possible to delete a topic, which relates to stories. 
It comes a 500 error from the api. We should make a proper exception out of it with the message, that it is not possible to delete topics, which are in use.

Currently, the frontend says nothing - the topic just remains without an error.